monkestation/code/datums/traits/neutral.dm

```diff
@@ -16,7 +16,8 @@
 
 /datum/quirk/anime
 	name = "Anime"
-	desc = "You are an anime enjoyer! Show your enthusiasm with some fashionable attire."
+	desc = "You are an anime enjoyer! You start the round with an Anime Dermal Implant kit, \
```

Comment:
Why has this been modified in this PR?

Reply:
It was vague, and this fit better with the description written for the prosthetic quirk.

Nearly every monke quirk has this problem, but I stopped there.

Reply:
You'll need to remove this one, since it is incorrect.
There is another PR directly above this one that will be adding more options, not just cat implants.